ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -54,13 +51,21 @@ const Prediction = (
     />
   ) : <Icon name="circle" size="small" color={color} />
   const fieldName = fieldTitle || snakecaseToTitlecase(field)
-  const fieldDisplay = dangerThreshold ? (
+  const fieldDisplay = thresholds ? (
     <Popup
       header={`${fieldName} Color Ranges`}
       content={
         <div>
-          <div>{`Red > ${dangerThreshold}`}</div>
-          {warningThreshold < dangerThreshold && <div>{`Yellow > ${warningThreshold}`}</div>}
+          {thresholds.map((th, i) => {
+            if (!th) {
+              return null
+            }
+            const t = pathHigher ? th : -1 * th
+            if (i < 3) {
+              return <div>{`${PATHOGENIC_COLORS[i]} ${pathHigher ? '<' : '>'}= ${t}`}</div>
+            }
+            return <div>{`${PATHOGENIC_COLORS[i]} ${pathHigher ? '>' : '<'}= ${t}`}</div>
```

Comment:
It simply shows the threshold and doesn't show the color ranges. Is it okay? And it still needs to align with the color display because I'm still determining what color we should use for both ends if thresholds are missing.